SpringParking-parkingBranch/src/main/java/pl/springDemo/demo/user/User.java

```diff
@@ -0,0 +1,54 @@
+package pl.springDemo.demo.user;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import pl.springDemo.demo.userPayment.UserPayment;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@Table(name = "User")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String login;
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private UserType userType;
+
+    @OneToMany(fetch = FetchType.LAZY , mappedBy = "userId", cascade = CascadeType.ALL)
+    private List<UserPayment> userPayment;
+
+    public User(String login, UserType userType) {
+        this.login = login;
+        this.userType = userType;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Czemu equals i hashCode nie jest generowany przez lomboka?

Reply:
Tak jak wcześniej pisałam, nie zdecydowałam się na to, ponieważ zdecydowałam delikatnie zkustomizować equals i hashCode